client/src/components/reader/Annotation/Selection/Truncated.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent, PropTypes } from 'react';
+import classNames from 'classnames';
+
+export default class AnnotationSelectionTruncated extends PureComponent {
+
+  static displayName = "Annotation.Selection.Truncated";
+
+  static propTypes = {
+    selection: PropTypes.string,
```

Comment:
This should be required, as the component will throw errors without a selection.